src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs

```diff
@@ -108,7 +119,12 @@ public override BlockState TryContinue(BlockProcessor processor, Block block)
 
         public override bool Close(BlockProcessor processor, Block block)
         {
-            var extensionName = ((TripleColonBlock)block).Extension.Name;
+			if ((block as TripleColonBlock).Extension.SelfClosing)
```

Comment:
Little rusty in C# but I think `(TripleColonBlock)block` style cast will lead to better error message- invalid cast exception instead of null reference exception.

In any case, the nit pick I came for was to suggest casting once and using the reference in this if-statement as well as the `var extensionName = ...` line that follows.